### PR TITLE
PE-6802: feat(timestamps)

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
- use ISO and UTC formats to display dates in the app

--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/1bedgacnqm31g

### CODE REVIEW

File: lib/pages/drive_detail/components/drive_detail_data_list.dart [hunk 1]
```diff
@@ -200,8 +200,8 @@ Widget _buildDataListContent(
           name: row.name,
           typography: typography,
           size: row.size == null ? '-' : filesize(row.size),
-          lastUpdated: yMMdDateFormatter.format(row.lastUpdated),
-          dateCreated: yMMdDateFormatter.format(row.dateCreated),
+          lastUpdated: row.lastUpdated,
```
Comment (karlprieb) [inline lib/pages/drive_detail/components/drive_detail_data_list.dart:203]: Is this intentional or you just forgot to use `formatDateToUtcString`?

**Comment by kunstmusik** [comment]
Did a search of the branch code and found yMMDateFormatter is still used in a few places and wonder if they need to be updated or if it is fine to leave as-is. If you could confirm @thiagocarvalhodev that'd be great.

image

File: lib/pages/drive_detail/components/drive_detail_data_list.dart (hunk 1 above)
Comment (thiagocarvalhodev) [reply lib/pages/drive_detail/components/drive_detail_data_list.dart:203]: Before we were passing a String, now we're passing the DateTime, and inside the component we convert them

**Comment by thiagocarvalhodev** [comment]
> Did a search of the branch code and found yMMDateFormatter is still used in a few places and wonder if they need to be updated or if it is fine to leave as-is. If you could confirm @thiagocarvalhodev that'd be great.
> 
> image

Good catch, @kunstmusik!! 

One of the files was not being used anywhere. I removed. There are just two places now on the item tile file. We will show on the drive explorer just the date instead of the timestamp, if the user hovers above the date we show the new formatted date

**Review by karlprieb — APPROVED** [review]

**Review by karlprieb — APPROVED** [review]

**Review by kunstmusik — APPROVED** [review]

**Pull request merged**